students/jesse_miller/session04/trigrams.py

```diff
@@ -4,10 +4,7 @@
 """
 import string
 import random
-#import os
-#import re
-import itertools
-
+import sys
 
 def parse_file():
```

Comment:
Modify _parse_file_ to take and use a filename, since you're asking the user for one :-)